tests/test_e2e.py

```diff
@@ -1036,5 +1040,54 @@ def test_acquire_token_silent_with_an_empty_cache_should_return_none(self):
         #       If this test case passes without exception,
         #       it means MSAL Python is not affected by that.
 
+
+@unittest.skipUnless(broker_available, "AT POP feature is only supported by using broker")
+class PopTestCase(LabBasedTestCase):
+    def test_at_pop_should_contain_pop_scheme_content(self):
+        auth_scheme = PopAuthScheme(
+            http_method="GET",
+            url="https://www.Contoso.com/Path1/Path2?queryParam1=a&queryParam2=b",
+            nonce="placeholder",
+            )
+        config = self.get_lab_user(usertype="cloud")
+        config["password"] = self.get_lab_user_secret(config["lab_name"])
+        result = self._test_username_password(auth_scheme=auth_scheme, **config)
```

Comment:
Consider calling `AcquireTokenSilent` + same `PopAuthScheme` after this and assert that the token does not come from the cache.

Reply:
Hmm, our latest PR of "token source" happens to allow us easily test whether a token came from cache. Would you mind also review and approve that PR? (`https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/610`)

Afterwards, I'll rebase this old POP PR to the latest `dev`, and then proceed to merge and ship it.